src/components/pipeline/ExecutionPipeline.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { withTranslation } from 'react-i18next';
+import { withRouter } from 'react-router';
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+
+import Main from '../common/Main';
+import BackButton from '../common/BackButton';
+
+import { EXECUTION_PIPELINE_BACK_BUTTON_ID } from '../../config/selectors';
+
+const styles = (theme) => ({
+  content: {
+    padding: theme.spacing(2),
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  backButton: {
+    position: 'fixed',
+    right: theme.spacing(2),
+    bottom: theme.spacing(2),
+  },
+});
+
+const ExecutionPipeline = (props) => {
```

Comment:
I would rather add the possibility to execute in `Executions` rather than in a whole new tab.  @hagoptaminian @Painguin

Reply:
That would make sense to me too. I.e. in the Executions page, you would still select a dataset, but have the option of selecting a Pipeline to run on it

Reply:
So for now maybe remove this page and icon